src/main/resources/main.css

```diff
@@ -113,6 +113,10 @@
     -fx-effect: dropshadow(gaussian, rgba(0, 0, 0, 0.5), 8, 0, 0, 5);
 }
 
+.avatar Label {
+    -fx-font-size: 36;
+    -fx-font-weight: bold;
```

Comment:
Did you check if that works? For me, it doesn't:
![image](https://cloud.githubusercontent.com/assets/3258762/17478543/e2fbc248-5d6c-11e6-93b3-f6858b38155e.png)
both avatar letters look very small